diagnostic.md

```diff
@@ -98,11 +127,25 @@ JSON.
 ```
 
 ```sh
-# your answer here
+curl --include --request "PATCH" "http://www.example.com/donuts/42" \
+--header "Content-Type: application/json" \
+--data $"{
+  \"donut\": {
+    \"id\": \"42\",
```

Comment:
You don't have to pass the id here. That is passed into the URL. But everything else is right.